src/components/HelFormFields/HelCheckbox.js

```diff
@@ -8,18 +8,15 @@ import Input from 'react-bootstrap/lib/Input.js'
 import {connect} from 'react-redux'
 import {setData} from 'src/actions/editor.js'
 
-let HelCheckbox = React.createClass({
+class HelCheckbox extends React.Component {
+    constructor(props) {
+        super(props)
 
-    contextTypes: {
-        intl: PropTypes.object,
-        dispatch: PropTypes.func
-    },
-
-    propTypes: {
-        name: PropTypes.string
-    },
-
-    handleCheck: function (event) {
+        this.handleCheck = this.handleCheck.bind(this)
+        this.getValue = this.getValue.bind(this)
```

Comment:
I dont think `getValue()` is even used so the method and binding could be removed.

Reply:
Do you think it's gonna be safer to have it there? Beware of the `this`